integration-test/helpers/pages/loginPage.js

```diff
@@ -279,12 +295,19 @@ export function loginPage (page, opts = {}) {
         async assertNoAttributesWereAdded () {
             const attrCount = page.locator('[data-ddg-inputtype]')
             const count = await attrCount.count()
-            expect(count).toBe(0)
+            await expect(count).toBe(0)
```

Comment:
Is this `await` needed? We're already awaiting `count`. 🤔

Reply:
Oh yeah! Not here